src/index.ts

```diff
@@ -12,6 +17,7 @@ export type RawValue = Value | Sql;
  * A SQL instance can be nested within each other to build SQL strings.
  */
 export class Sql {
+  readonly bindParams = 0;
```

Comment:
The only question to myself is whether `values` should now become a getter that throws when `bindParams > 0`? That would ensure the instance isn't incorrectly used without binding params.